client/main.lua

```diff
@@ -2,6 +2,7 @@ local Cfg = Cfg
 local currentGrid = 0
 -- we can't use GetConvarInt because its not a integer, and theres no way to get a float... so use a hacky way it is!
 local volume = tonumber(GetConvar('voice_defaultVolume', '0.3'))
+local radioVolume = volume
```

Comment:
might want to make a 'volumes' table so there can be a 'phone' and 'radio' getters/setters.

Reply:
Properly a more long-term proof idea, yea.